lib/glob-assets.js

```diff
@@ -42,7 +41,7 @@ module.exports = async ({cwd}, assets) =>
                 // - `filepath` ignored (also to avoid duplicates)
                 // - other properties of the original asset definition
                 const {filepath, ...others} = asset;
-                return globbed.map(file => ({...others, path: file, label: basename(file)}));
+                return globbed.map(file => ({...others, path: file, label: path.relative(pkgRoot || '.', file)}));
```

Comment:
Should we consider this a breaking change?

Reply:
@fgreinacher I wouldn't unless generating duplicate labels was support somewhere and things depended o nit. Honestly don't know. Seems weird. Imagine creating a listing of files in multiple directories and losing the directory they were in, that was the old behavior.  So the labels would collide and the SCM would reject the operation.